modules/core/src/v2/coinFactory.ts

```diff
@@ -68,8 +69,15 @@ import { Bcha } from './coins/bcha';
 import { Tbcha } from './coins/tbcha';
 import { Dot } from './coins/dot';
 import { Tdot } from './coins/tdot';
+import { Environments } from './environments';
 
 export type CoinConstructor = (bitgo: BitGo, staticsCoin?: Readonly<StaticsBaseCoin>) => BaseCoin;
+const coinConstructorDual =
```

Comment:
The name could not help in understand the use, it is a "Environment Dependent Coin Constructor", at least a comment can help readers understand "why" it is needed

Reply:
done